openstack/identity/v3/projects/requests.go

```diff
@@ -243,3 +243,30 @@ func Update(ctx context.Context, client *gophercloud.ServiceClient, id string, o
 	_, r.Header, r.Err = gophercloud.ParseResponse(resp, err)
 	return
 }
+
+// GetProjectTag checks if a a tag exists in a project.
+func GetProjectTag(ctx context.Context, client *gophercloud.ServiceClient, projectID string, tag string) (r GetProjectTagResult) {
+	resp, err := client.Get(ctx, getProjectTagURL(client, projectID, tag), nil, &gophercloud.RequestOpts{
+		OkCodes: []int{204},
+	})
+	_, r.Header, r.Err = gophercloud.ParseResponse(resp, err)
+	return
+}
+
+// AddProjectTag adds a tag in a project.
+func AddProjectTag(ctx context.Context, client *gophercloud.ServiceClient, projectID string, tag string) (r AddProjectTagResult) {
+	resp, err := client.Put(ctx, addProjectTagURL(client, projectID, tag), nil, nil, &gophercloud.RequestOpts{
+		OkCodes: []int{201},
+	})
+	_, r.Header, r.Err = gophercloud.ParseResponse(resp, err)
+	return
+}
+
+// DeleteProjectTag deletes a tag from a project.
+func DeleteProjectTag(ctx context.Context, client *gophercloud.ServiceClient, projectID string, tag string) (r DeleteProjectTagResult) {
```

Comment:
Again, make this plural -> `DeleteTags()`.

Also, the API call drops all tags, and should not take a `tag` argument.

Reply:
For this one, we want to implement the `Delete a single tag from project` API call which take a single tag and deletes it and not the `Remove all tags from a project`. Therefore, `DeleteTag` is a more suitable name, and therefore I think we still need the tag argument.